elevate-user/constants/routes.js

```diff
@@ -22,6 +22,28 @@ module.exports = {
 				functionName: 'updateUser',
 			},
 		},
+		{
+			sourceRoute: '/interface/v1/account/reActivate',
+			type: 'POST',
+			inSequence: true,
+			orchestrated: true,
+			targetRoute: {
+				path: '/user/v1/account/reActivate',
+				type: 'POST',
+				functionName: 'reActivateAccount',
+			},
+		},
+		{
+			sourceRoute: '/user/v1/account/reActivateOtp',
+			type: 'POST',
+			inSequence: false,
+			orchestrated: false,
+			targetRoute: {
+				path: '/user/v1/account/reActivateOtp',
+				type: 'POST',
+				functionName: 'reActivateAccount',
```

Comment:
If its pass-through call why we need the function name @sumanvpacewisdom and you have not created a function as well for this

Reply:
@ankitpws got it and i have made this changes accordingly.